Meta/check-style.py

```diff
@@ -67,20 +78,28 @@ def find_files_here_or_argv():
     return filter(should_check_file, raw_list)
 
 
+def is_forbidden(filename, forbidden_list):
```

Comment:
"forbidden" requires a bit of imagination, maybe call this something like `list_contains_filename` or `filename_matches_list`?

Reply:
You're right! I just copied the name from the constant without reflecting on it. I think I'll call it `is_in_prefix_list`, because the "prefix" aspect is important in my eyes.